server/src/main/java/org/tbbtalent/server/service/db/impl/BrandingServiceImpl.java

```diff
@@ -34,15 +36,17 @@
  * @author John Cameron
  */
 @Service
+@RequiredArgsConstructor
 public class BrandingServiceImpl implements BrandingService {
     private final PartnerService partnerService;
     private final UserService userService;
 
-    public BrandingServiceImpl(PartnerService partnerService, UserService userService) {
-        this.partnerService = partnerService;
-        this.userService = userService;
-    }
-
+    /**
+     * Returns the branding information for a partner.
```

Comment:
We don't duplicate JavaDoc on method implementations that is already present on the method as described in the interface. That leads to duplication which easily gets out of sync.

Reply:
Yep - makes sense - removed.